src/app/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Button from './Button';
+
+export default {
+  title: 'Component/Button',
+  component: Button,
+};
+
+export const Regular = (): JSX.Element => (
+  <Button onClick={() => console.log('clicked!')}>Schüler hinzufügen</Button>
```

Comment:
I think you can remove this onClick function, if it only logs something in the console for control purposes during development.

Reply:
I really would like to, but TypeScript needs it to be happy!